torch_ttnn/patterns/add.py

```diff
@@ -0,0 +1,24 @@
+import torch
+import ttnn
```

Comment:
why is this actually needed?

Reply:
Sorry, I made a comment on this earlier but didn't realize I need to confirm one more step before it shows up. It doesn't seem like this file is being used at all, so I don't think this is needed.

Reply:
@yocox @yocosky, I once heard that we need two patterns to compile `x + y`, one for `x` = `y` and the other for `x` ≠ `y`. Is that still the case?

Reply:
@ayerofieiev-tt @kevinwuTT This file is not needed anymore. I left the file as the example of 2 things:
1. How to use the sub-graph rewriter
2. How to support when some input tensors are the same tensor.

The file is not used actually and since you all had seen this file, I think it can be removed.

Reply:
Please remove. Thank you